services/background-worker/background-worker.cabal

```diff
@@ -148,6 +159,96 @@ executable background-worker
     UndecidableInstances
     ViewPatterns
 
+executable background-worker-integration
```

Comment:
ah, no, wait.  please don't introduce any new legacy integration test suites.  we're using /integration for integration tests from now on; `/services/<package>/test/integration` is only still there because it takes time and effort to migrate.

another thing: previously the unit tests and integration tests should be in separate directories, and do separate things.  like in `/services/brig/test/{unit,integration}`.  only the `integration` part is now legacy, and we use `/integration` with a new approach instead.  unit tests in any package are fine, of course.

please pair when you're using `/integration` for the first time!  it has a readme, but it's a bit dense.

Reply:
I'll do some prep reading before you come back online.

Reply:
what happened to this?